### DISCUSSION TITLE
using   ts-patch-mongoose with typegoose

### DISCUSSION THREAD

**Discussion by nourissam** [ilovepixelart/ts-patch-mongoose#26] [Q&A] [answered]
Can we use ts-patch-mongoose with typegosse class??

**Comment by ilovepixelart** [comment, marked as answer]
Yes you should be able to use it, since this is a plugin for mongoose
But if you talking about decorators
https://typegoose.github.io/typegoose/docs/api/decorators/plugin/
Try following this example, if it doesn't work, try creating a simple repo with example (post link here) I will take a look
